docs/source/tutorials_source/package/tutorial_pretrain_detectron2.py

```diff
@@ -0,0 +1,254 @@
+"""
+.. _lightly-detectron-tutorial-6:
+
+.. role:: bash(code)
+   :language: bash
+
+Tutorial 6: Pre-train a Detectron2 Backbone with Lightly
+==========================================================
+
+In this tutorial we show how you can do self-supervised pre-training of a
+Detectron2 backbone with lightly. The focus of this tutorial is on how to get
+and store a pre-trained ResNet50 backbone of the popular Detectron2 framework.
+If you want to learn more about self-supervised learning in general, go check
+out the following tutorials:
+
+ - :ref:`lightly-moco-tutorial-2`
+ - :ref:`lightly-simclr-tutorial-3`:
+ - :ref:`lightly-simsiam-tutorial-4`
+
+What you will learn:
+
+- How to retrieve a Detectron2 ResNet50 backbone for pre-training
+- How to do self-supervised learning with the backbone
+- How to store the backbone in a checkpoint file which can be used by Detectron2
+
+Prerequisites:
+----------------
+For the purpose of this tutorial you will need to:
+
+- Install Lightly: Follow the `instructions <https://docs.lightly.ai/getting_started/install.html>`_.
+- Install Detectron2: Follow the `instructions <https://detectron2.readthedocs.io/en/latest/tutorials/install.html>`_.
+- Download a dataset for pre-training (we will use the `SKU-110K <https://github.com/eg4000/SKU110K_CVPR19>`_ dataset).
```

Comment:
Maybe mention the direct link to download the dataset we use? `http://trax-geometry.s3.amazonaws.com/cvpr_challenge/SKU110K_fixed.tar.gz`